API.md

```diff
@@ -63,3 +66,49 @@ The url is in the form:
 #### Example:
 
 http://geojson.io/#id=github:benbalter/dc-wifi-social/blob/master/bars.geojson&map=14/38.9140/-77.0302
+
+## Console API
+
+Pop open your browser console and see the beautiful examples: geojson.io has started to expose a subset of its inner workings for you to mess around with:
+
+
+### `window.api.map`
+
+The [Mapbox GL JS](https://docs.mapbox.com/mapbox-gl-js/guides/) map that you see and use on the site. See the [Mapbox GL JS API Reference](https://docs.mapbox.com/mapbox-gl-js/api/) for all the things you can do with it.
+
+For instance, you could add another map source and layer:
+
+```js
+window.api.map.addSource('raster-tiles', {
+    'type': 'raster',
+    'tiles': [
+    'https://stamen-tiles.a.ssl.fastly.net/watercolor/{z}/{x}/{y}.jpg'
+    ],
+    'tileSize': 256,
+    'attribution':
+    'Map tiles by <a target="_top" rel="noopener" href="http://stamen.com">Stamen Design</a>, under <a target="_top" rel="noopener" href="http://creativecommons.org/licenses/by/3.0">CC BY 3.0</a>. Data by <a target="_top" rel="noopener" href="http://openstreetmap.org">OpenStreetMap</a>, under <a target="_top" rel="noopener" href="http://creativecommons.org/licenses/by-sa/3.0">CC BY SA</a>'
+})
+
+window.api.map.addLayer({
+    'id': 'simple-tiles',
+    'type': 'raster',
+    'source': 'raster-tiles',
+    'minzoom': 0,
+    'maxzoom': 22
+})
```

Comment:
This is a fabulous example, though we should be alert to [the rumblings](https://stamen.com/30daymapchallenge-2022/) about Stamen shutting down their tile endpoints.

<img width="1439" alt="image" src="https://user-images.githubusercontent.com/735463/216098393-a6abd9fc-32df-4b95-8de0-246c02e03183.png">

Reply:
Noted, thanks for bringing that up... we can swap it out later I think.